src/plugins/data_source_management/public/components/direct_query_data_sources_components/acceleration_creation/selectors/preview_sql_defintion.tsx

```diff
@@ -60,24 +62,8 @@ export const PreviewSQLDefinition = ({
   };
 
   const checkIfSQLWorkbenchPluginIsInstalled = () => {
-    fetch('/api/status', {
-      headers: {
-        'Content-Type': 'application/json',
-        'osd-xsrf': 'true',
-        'accept-language': 'en-US,en;q=0.9,zh-CN;q=0.8,zh;q=0.7,zh-TW;q=0.6',
-        pragma: 'no-cache',
-        'sec-fetch-dest': 'empty',
-        'sec-fetch-mode': 'cors',
-        'sec-fetch-site': 'same-origin',
-      },
-      method: 'GET',
-      referrerPolicy: 'strict-origin-when-cross-origin',
-      mode: 'cors',
-      credentials: 'include',
-    })
-      .then(function (response) {
-        return response.json();
-      })
+    http
```

Comment:
Nit: we'd better add unit test for this line.

Reply:
Sure, and I have added a test class for this entire class.